test/controllers/secure.register.filter.controller.spec.ts

```diff
@@ -4,11 +4,15 @@ jest.mock('../../src/middleware/authentication.middleware');
 jest.mock('../../src/utils/application.data');
 jest.mock('../../src/middleware/navigation/has.sold.land.middleware');
 jest.mock("../../src/middleware/service.availability.middleware");
+jest.mock("../../src/middleware/navigation/remove/remove.journey.middleware");
 
 import { NextFunction, Request, Response } from "express";
 import { beforeEach, expect, jest, test, describe } from "@jest/globals";
 import request from "supertest";
 
+// import remove journey middleware mock before app to prevent real function being used instead of mock
+import { removeJourneyMiddleware } from "../../src/middleware/navigation/remove/remove.journey.middleware";
```

Comment:
Comment refers to mock whereas this looks to be importing the real function..

Reply:
jest.mock("../../src/middleware/navigation/remove/remove.journey.middleware"); replaces real implementation of function

Reply:
Looks like the comment has been removed (which I'm happy with) but it's present in 34 other test files, so should really be removed in those too. @dlloyd1-ch @markpit agree?

Reply:
Aha. Thanks for clarifying.

Could we make it consistent and import the mock here as well, rather than doing the mocking on lines 49 and 50 I wonder? Would mean the comment goes back in! Is this a crazy idea?!

Reply:
Happy to resolve.